File: momentra/momentra/Shell/GroupEmpty/GroupInviteLink.swift
import Foundation

enum GroupInviteLink {
    private static let httpsBase = "https://momentra.app/j"

    static func displayPath(code: String) -> String {
        "\(httpsBase)/\(code.trimmingCharacters(in: .whitespacesAndNewlines).lowercased())"
    }

    static func copyText(code: String) -> String {
        displayPath(code: code)
    }

    /// Same HTTPS URL as share/copy so camera apps and messengers hit the invite landing.
    static func qrPayload(code: String) -> String {
        displayPath(code: code)
    }
}

File: apk/app/src/main/java/com/example/momentra/ui/shell/empty/group/GroupInviteLink.kt
package com.example.momentra.ui.shell.empty.group

internal object GroupInviteLink {
    private const val HTTPS_BASE = "https://momentra.app/j"

    fun displayPath(code: String): String = "$HTTPS_BASE/${code.trim().lowercase()}"

    fun copyText(code: String): String = displayPath(code)

    /** Same HTTPS URL as share/copy so camera apps and messengers hit the invite landing. */
    fun qrPayload(code: String): String = displayPath(code)
}
